egs/swbd/s5c/local/nnet3/tuning/run_tdnn_b.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# b is as a, but using a different splice indexes and geometrically increasing
+# relu dims across layers. It gives better results but takes 2.5x time for training
+
+# System                     a           b
+# WER on train_dev(tg)    17.4        16.0       
+# WER on train_dev(fg)    16.0        14.9
+# WER on eval2000(tg)     19.7        18.5
+# WER on eval2000(fg)     18.0        17.1
+# WER on rt03(tg)         22.6        21.6
+# WER on rt03(fg)         20.0        19.1
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=0
+affix=
+train_stage=-10
+has_fisher=true
+speed_perturb=true
+common_egs_dir=
+reporting_email=
+remove_egs=true
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+suffix=
+if [ "$speed_perturb" == "true" ]; then
+  suffix=_sp
+fi
+dir=exp/nnet3/tdnn_b
+dir=$dir${affix:+_$affix}
+dir=${dir}$suffix
+train_set=train_nodup$suffix
+ali_dir=exp/tri4_ali_nodup$suffix
+
+local/nnet3/run_ivector_common.sh --stage $stage \
+	--speed-perturb $speed_perturb || exit 1;
+
+if [ $stage -le 9 ]; then
+  echo "$0: creating neural net configs";
+
+  # create the config files for nnet initialization.
+  # Note: --relu-dim-init is the dim for the first relu layer, and --relu-dim is
+  # the dim for the last relu layer, and the dims of the layers in between increase
+  # geometrically. The reason for this kind of parameters re-distribution is mainly
+  # for time efficiency, since the lower layers are more computationally intensive.
+  steps/nnet3/tdnn/make_configs.py  \
+    --feat-dir data/${train_set}_hires \
```

Comment:
OK, I didn't realize that your baseline was the 4 layer network. Most of the increase in training time probably just due to the doubling of number of layers (4 sequential computations vs 8 sequential computations), and there is no way to avoid this. I wouldn't worry about the increase in the training time any more. 

I would just request you to compare the real-time factors of the two models, during decoding.